src/layouts/SiteLaunchPad/components/SiteLaunchCheckList.tsx

```diff
@@ -124,38 +130,44 @@ const addSubtitlesForChecklist = (
   TABLE_MAPPING: TableMappingProps[],
   tasksDone: number
 ): TableMappingProps[] => {
-  const newTableMapping = [...TABLE_MAPPING]
-  newTableMapping[SITE_LAUNCH_TASKS.SET_DNS_TTL - 1].subTitle = (
-    <>
-      {tasksDone === SITE_LAUNCH_TASKS.SET_DNS_TTL - 1 && (
-        <Text fontSize="small">
-          You can check you current DNS TTL through 3rd party applications such
-          as <Link href="https://www.nslookup.io/">nslookup.io</Link>
-        </Text>
-      )}
-    </>
-  )
-  newTableMapping[SITE_LAUNCH_TASKS.DROP_CLOUDFRONT - 1].subTitle = (
-    <>
-      {tasksDone === SITE_LAUNCH_TASKS.DROP_CLOUDFRONT - 1 && (
-        <Text fontSize="small">
-          If you are using CWP, please contact them to do this for you
-        </Text>
-      )}
-    </>
-  )
-  newTableMapping[
-    SITE_LAUNCH_TASKS.DELETE_EXISTING_DNS_RECORDS - 1
-  ].subTitle = (
-    <>
-      {tasksDone === SITE_LAUNCH_TASKS.DELETE_EXISTING_DNS_RECORDS - 1 && (
-        <Text fontSize="small">
-          This should be done as soon as domains are dropped
-        </Text>
-      )}
-    </>
-  )
-  return newTableMapping
+  return TABLE_MAPPING.map((task) => {
```

Comment:
mainability: this is not a functionality concern, but having this code structure makes it difficult to decipher what this code chunk does. the reason for this is because don't directly map into a new component but return a component as part of an obj as part of an array, which is then consumed somewhere.

would be good if this returned a plain object but because this would qualify as a refactor, am ok with filing as ticket + add `TODO` + comment

Reply:
sure 6f6f7f4f1114bb40d81808192ad7005381a21ded

Reply:
actually for additional clarification, what i meant here is that this `maps` and returns an object where `subTitle` is a jsx component and **not** a value.

this mixes in render (the component) together with the value (the text shown by the string). is this a big effort to chaneg?

Reply:
Correct me if I am wrong here, but yes, because `subtitle` is not a `string`! 
Reason for this is that the subtitles have things like links!